composer.json

```
@@ -61,18 +66,19 @@
     }
   ],
   "require-dev": {
+    "php": "^7.4 || ^8.0",
     "alleyinteractive/alley-coding-standards": "^0.3.0",
+    "alleyinteractive/pest-plugin-wordpress": "dev-main",
```

You can track the new version now!

```suggestion
    "alleyinteractive/pest-plugin-wordpress": "^0.4.0",
```